Source/Project64/Multilanguage/Language Class.cpp

```diff
@@ -269,13 +270,16 @@ void CLanguage::LoadDefaultStrings (void)
 	DEF_STR(ROM_USE_TLB,         L"Use TLB");
 	DEF_STR(ROM_REG_CACHE,       L"Register caching");
 	DEF_STR(ROM_DELAY_SI,        L"Delay SI Interrupt");
-	DEF_STR(ROM_SP_HACK,         L"SP Hack");
+	DEF_STR(ROM_VIREFRESH,       L"Vi Refresh Rate: ");

```

Comment:
There wasn't enough space to add new strings, so I decided to remove one that was unused. Since the UI doesn't have any string called "SP Hack", I knew it was safe to remove